codec/decoder/core/src/decode_slice.cpp

```diff
@@ -56,6 +56,22 @@ extern void FreePicture (PPicture pPic, CMemoryAlign* pMa);
 
 extern PPicture AllocPicture (PWelsDecoderContext pCtx, const int32_t kiPicWidth, const int32_t kiPicHeight);
 
+static bool CheckRefPics (const PWelsDecoderContext& pCtx) {
+  int32_t listCount = 1;
```

Comment:
So it is only for P frame decoding? Why it occurs now since P frame is always on?